src/utils/rest-api.js

```diff
@@ -2428,6 +2424,59 @@ export function fetchMapEquipments(
     return backendFetchJson(fetchEquipmentsUrl);
 }
 
+export function fetchSingleEquipment(
+    studyUuid,
+    currentNodeUuid,
+    equipmentId,
+    equipmentResource,
+    inUpstreamBuiltParentNode
+) {
+    console.info(
+        `Fetching '${equipmentResource}' '${equipmentId}' from '${studyUuid}' and node '${currentNodeUuid}'...`
+    );
+    let urlSearchParams = new URLSearchParams();
+    urlSearchParams.append(
+        'inUpstreamBuiltParentNode',
+        inUpstreamBuiltParentNode
+    );
+    let fetchEquipmentsUrl =
+        getStudyUrlWithNodeUuid(studyUuid, currentNodeUuid) +
+        '/network-map/' +
+        equipmentResource +
+        '/' +
+        equipmentId +
+        '?' +
+        urlSearchParams.toString();
+
+    console.debug(fetchEquipmentsUrl);
+    return backendFetchJson(fetchEquipmentsUrl);
+}
+
+export function fetchBranchStatus(
```

Comment:
You didn't want to use fetchEquipmentInfos ?
It is not appropriate ?

Reply:
I can use fetchEquipmentInfos on 2WT, but in this case I have to add getVoltageLevel().getName() on terminal1 & terminal2
I'm gonna try this.